lib/msal-common/src/logger/Logger.ts

```diff
@@ -62,14 +62,25 @@ export class Logger {
         const defaultLoggerCallback = () => {
             return;
         };
-        this.localCallback = loggerOptions.loggerCallback || defaultLoggerCallback;
-        this.piiLoggingEnabled = loggerOptions.piiLoggingEnabled || false;
-        this.level = typeof(loggerOptions.logLevel) === "number" ? loggerOptions.logLevel : LogLevel.Info;
-        this.correlationId = loggerOptions.correlationId || Constants.EMPTY_STRING;
+        const setLoggerOptions = loggerOptions || Logger.createDefaultLoggerOptions();
```

Comment:
Let's change `LoggerOptions` in the constructor to `Required<LoggerOptions>` and then you don't need to repeat the default logic here

Reply:
This will entail type changes in ClientApplication.ts for msal-node. @tnorling